Enterprise-CMCS/managed-care-review
# MCR-4202: Generate Download Links for Documents

## Summary

Adds a `downloadURL` field to `GenericDocuments` (used by contracts and rates) and `Documents` (used by questions)

#### Related issues
https://jiraent.cms.gov/browse/MCR-4202

#### Screenshots

#### Test cases covered

- `downloadURL` is populated on fetch/index for GenericDocument and Documents

## QA guidance

NOTE: if testing this locally you'll have to upload new files in order to test this. Files uploaded via local tests will not be valid

GenericDocument (used by rates and contracts)
- Upload new files to a submission
- View a page that calls fetchContract or fetchRate 
- Using the network tab, grab the `downloadURL` from formData
- Inputting the url should trigger the download for the file

Questions
- Upload a new question
- Using the network tab, grab the `downloadURL` from formData
- Inputting the url should trigger the download for the file
